components/Layout.tsx

```diff
@@ -152,10 +152,25 @@ const MainNavigation = () => {
   useEffect(() => {
     const icon = theme === 'dark' ? 'herobtn' : '';
     const menu = theme === 'dark' ? 'bg-white' : 'bg-black';
+    const dataTheme = theme === 'dark' ? 'dark' : 'light';
+    const keyShadow =
```

Comment:
How this logic differs from the one in DarkModeToggle.tsx?

Reply:
Actually we only need it here in Layout file not in DarkModeToggle.tsx, I have fixed it now.